### PR TITLE
741 create a setting screen

### PR DESCRIPTION
**Pull request by ismaelAbogabal**
Add setting screen 
Move all user preferences into this screen 
So the user can manage all his preferences  from a single place and we can explain to him what each option means 

image

### CODE REVIEW

**Comment by ibrahim-zehhaf-mawaqit** [comment]
please fix conflict

**Comment by ibrahim-zehhaf-mawaqit** [comment]
please fix conflict

**Comment by ismaelAbogabal** [comment]
@ibrahim-zehhaf-mawaqit Conflict resolved

> please fix conflict

File: lib/l10n/intl_en.arb [hunk 1]
```diff
@@ -141,10 +141,13 @@
...
-  "announcementOnlyModeExplanation": "Will show announcements all time",
+  "secondaryScreenExplanation": "Secondary mosque room. this screen made for the secondary screen in the women room or any other room except the main mosque room. this screen will has slightly different behavior than the main screen. this screen will show the jumuaa live stream during jumuaa if there are any.",
+  "mainScreenExplanation": "Main mosque room. this screen made for the main screen in the mosque. this screen will show the jumuaa live stream during jumuaa if there are any.",
+  "normalModeExplanation": "App will show the normal screen with the salah times and the announcements. all the invocations will be triggered according to the mosque configurations on the backend server.",
+  "announcementOnlyModeExplanation": "App will show only the announcements screen with the salah times. this is useful for the mosques that want to use this as a secondary screen to show the announcements and salah times only.",
+  "settings": "Settings",
+  "applicationModes": "Application Modes",
+  "ifYouAreFacingAnIssueWithTheAppActivateThis" : "If you are facing issues with the app, try to enable this option",
```
Comment (ibrahim-zehhaf-mawaqit) [inline lib/l10n/intl_en.arb:150]: Please keep the old translations

**Comment by ismaelAbogabal** [comment]
@ibrahim-zehhaf-mawaqit The translations are reverted back. In this ee5261211baf80c156070ba70bb34679885170f7 commit.
Please check it out.

**Review by ibrahim-zehhaf-mawaqit — APPROVED** [review]

**Pull request merged**